CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+Version 4.3.2
+=============
+
+Features
+--------
+
+* TRC 1.3 Support: Added support for .trc log files as generated by PCAN Explorer v5 and other tools, expanding compatibility with common log file formats (#1753).
+* ASCReader Performance: Significantly improved the read performance of ASCReader, optimizing log file processing and analysis (#1717).
```

Comment:
It's probably not much of a difference in performance. The code is a little bit cleaner though.